src/simulations/Simulations.jl

```diff
@@ -100,12 +101,19 @@ function LandSimulation(
         ),
     ),
 )
-    if !isnothing(diagnostics) &&
-       !isempty(diagnostics) &&
-       first(diagnostics).output_writer.output_dir != outdir
-        @warn "Note that the kwarg outdir and outdir used in diagnostics are inconsistent; using $(first(diagnostics).output_writer.output_dir)"
+    if !isnothing(diagnostics) && !isempty(diagnostics)
+        try
+            ow = first(diagnostics).output_writer.output_dir
```

Comment:
this is required since the NetCDFWriter has an output dir (writes to file), while the DictWriter does not.